app/index.html

```diff
@@ -19,13 +19,25 @@
     <link rel="apple-touch-icon" href="assets/images/goodcity-icon-57.png" />
     <link rel="apple-touch-icon" sizes="72x72" href="assets/images/goodcity-icon-72.png" />
     <link rel="apple-touch-icon" sizes="114x114" href="assets/images/goodcity-icon-114.png" />
+    <script type="text/javascript">
+      document.addEventListener("deviceready", onDeviceReady, false);
+
+      function onDeviceReady(){
+          StatusBar.overlaysWebView(false);
+          StatusBar.backgroundColorByName("black");
+          StatusBar.styleLightContent();
+      }
+    </script>
 
     {{content-for 'head-footer'}}
 
   </head>
   <body>
     {{content-for 'body'}}
     <script src="assets/vendor.js"></script>
+    <script src="http://builds.emberjs.com/tags/v2.3.1/ember.prod.js"></script>
```

Comment:
what effect do these lines have on the mobile app? Do they also download the external content? It would be best to ship them as part of the assets folder.

Reply:
I was testing something using that. I think it is no longer needed. Thanks for pointing @steveyken . I have removed it.